### PR TITLE
[UI] Replace text field by MarkdownTextArea

### LINKED ISSUE #1387: [UI] Replace text field by MarkdownTextArea
**Reserved** for [outreachy](https://www.outreachy.org/) candidates. In order for this to be assigned to you, make sure to complete the first step of the application process outlined in the [project description](https://www.outreachy.org/december-2019-to-march-2020-internship-round/communities/mozilla/#build-a-component-that-can-display-a-json-schema).

---

This is the exact same issue as https://github.com/taskcluster/taskcluster/issues/1363 but for the description fields in
* https://taskcluster-ui.herokuapp.com/auth/roles/create
* https://taskcluster-ui.herokuapp.com/auth/clients/create
* https://taskcluster-ui.herokuapp.com/auth/roles/hook-id%3Ahg-push%2Fmozilla-esr60
* https://taskcluster-ui.herokuapp.com/auth/clients/email%2Fchinmaykousik1%40gmail.com%2Fgeneric-worker-test

### PR DESCRIPTION
**Pull request by Rolikasi**
Import MarkdownTextArea, replace Texfield in description fields with MarkdownTextArea in RoleForm and ClientForm

Finally I got rid of previous commits on master branch. Now I really understand how to work with branches!

Closes #1387 

the same in roles/create , roles/... and clients/...
solved_3 1_issue

### CODE REVIEW

**Comment by djmitche** [comment]
Looks good to me! @helfi92 will be back next week, and will offer his review at that time.

File: ui/src/components/RoleForm/index.jsx [hunk 1]
```diff
@@ -192,14 +193,11 @@ export default class RoleForm extends Component {
             </Fragment>
           )}
           <ListItem>
-            <TextField
-              label="Description"
-              name="description"
+            <MarkdownTextArea
               onChange={this.handleInputChange}
```
Comment (helfi92) [inline ui/src/components/RoleForm/index.jsx]: There seems to be a bug in the onChange handler.

STR:

1. Navigate to `https://deploy-preview-1447--taskcluster-web.netlify.com/auth/roles/github-admin%3A*` (deploy preview of your latest commit)
2. Click on the "Write" tab
3. Try to change the text

Expected: The text can be edited.
Actual: The text cannot be edited.

File: ui/src/components/RoleForm/index.jsx [hunk 2]
```diff
@@ -192,14 +193,11 @@ export default class RoleForm extends Component {
...
-              fullWidth
-              multiline
-              rows={5}
+              placeholder="Role form description (markdown)"
```
Comment (helfi92) [inline ui/src/components/RoleForm/index.jsx]: "Role description (markdown)"

File: ui/src/components/ClientForm/index.jsx [hunk 3]
```diff
@@ -324,14 +325,11 @@ export default class ClientForm extends Component {
...
           <ListItem>
-            <TextField
-              label="Description"
-              name="description"
+            <MarkdownTextArea
               onChange={this.handleInputChange}
-              fullWidth
-              multiline
-              rows={5}
+              placeholder="Client form description (markdown)"
```
Comment (helfi92) [inline ui/src/components/ClientForm/index.jsx]: "Client description (markdown)"

**Comment by Rolikasi** [comment]
@helfi92 I fixed this issue, handleInputChange didn't receive the name from MarkDownTextArea. Ready for a review!
Thank you!)

**Pull request closed without merging**